kaldb/src/test/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryServiceTest.java

```diff
@@ -177,12 +178,17 @@ private static Server newIndexingServer(
       KaldbTimeoutLocalQueryService wrapperService =
           new KaldbTimeoutLocalQueryService(service, waitForSearchMs);
       return Server.builder()
+          .workerGroup(
+              EventLoopGroups.newEventLoopGroup(4, "armeria-common-worker-indexer-delayed", true),
```

Comment:
instead of 4 should we do the formula?

Reply:
I'd prefer if we keep it a constant, so it behaves more consistently between CI and localhost.

Reply:
fair :) We've been bitten by it.. haha